api/src/main/java/cz/metacentrum/perun/spRegistration/persistence/configs/AppConfig.java

```diff
@@ -254,6 +255,17 @@ public void setPerunAttributeDefinitionsMap(Map<String, PerunAttributeDefinition
 		this.perunAttributeDefinitionsMap = perunAttributeDefinitionsMap;
 	}
 
+	public Map<String, String> getMails() { return mails; }
```

Comment:
Change "mails" to something more descriptive.  We are using this map as a register for the institution -> authority email pairing, when user requests moving service into the production environment.
I would suggest something like prodTransferAuthoritiesMailsMap

Reply:
Done